data-node/src/main/java/org/graylog/datanode/configuration/variants/OpensearchSecurityConfiguration.java

```diff
@@ -133,6 +144,29 @@ public Map<String, String> getProperties() throws GeneralSecurityException, IOEx
         return config.build();
     }
 
+    private Map<String, Object> getMap(final Map<String, Object> map, final String... keys) {
```

Comment:
The method name is too generic, maybe name it sth like. `filterMap`?

Reply:
renamed